test/s3Helper.js

```diff
@@ -16,6 +16,16 @@ test('s3Helper', (t) => {
     }
   })
 
+  t.test('extractPseudoKey', (t) => {
+    t.plan(2)
+    const s3Prefix = '0/xGrUe8vokl9kjAx+RTu9t6I1UnOT7mcdcizAVI+2bos=/2/1482435340000'
+    const data = new Uint8Array(10)
+    const s3Keys = s3Helper.encodeDataToS3KeyArray(s3Prefix, data)
+    t.equal(s3Keys.length, 1, `${t.name} encodes to one part`)
+    const pseudoKey = s3Helper.extractPseudoKey({ Key: s3Keys[0] })
+    t.equal(pseudoKey, '1482435340000-3ZYcXs', `${t.name} expected pseudo-key`)
```

Comment:
is it possible to get `3ZYcXs` by calculating it like https://github.com/brave/sync/pull/306/files#diff-33e6fea82e3ab87ec881daae3c0c6384R41 ?

Reply:
Thanks for pointing. Fixed.